examples/python/awq-quantized-model.py

```diff
@@ -130,9 +130,11 @@ def main():
 
     create_model(model_name, input_folder, output_folder, precision, execution_provider, cache_dir, **extra_options)
 
+    if args.execution_provider == "dml":
+        if og.__id__ != "onnxruntime-genai-directml":
+            raise ValueError(f"onnxruntime-genai-directml is required to be installed.")
```

Comment:
Can we clarify that all ONNX Runtime GenAI packages should be uninstalled and only the DML version should be installed?

```suggestion
            raise ValueError(f"onnxruntime-genai-directml is required to be installed. Please uninstall all ORT GenAI packages with `pip uninstall -y onnxruntime-genai onnxruntime-genai-cuda onnxruntime-genai-directml` and only install the DML version with `pip install onnxruntime-genai-directml`.")
```